### PR TITLE
Feat/telecom order

### PR DESCRIPTION
**Pull request by marie-j**
ref: MANAGER-4206

### CODE REVIEW

File: packages/manager/modules/sms/src/sms/sms.routing.js [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+        template: smsView,
+        controller,
+        controllerAs: 'TelecomSmsCtrl',
+      },
+    },
+    abstract: true,
+    resolve: {
+      initSms: ($q, $stateParams, TucSmsMediator) => {
+        // init sms services
+        TucSmsMediator.initAll().then((smsDetails) => TucSmsMediator
```
Comment (FredericEspiau) [inline packages/manager/modules/sms/src/sms/sms.routing.js]: You don't chain ? To make sure stufss are initialized before saying they are

Comment (marie-j) [reply packages/manager/modules/sms/src/sms/sms.routing.js]: Actually I just extracted the existing routing, some rework might be needed in the future 👀

**Comment by ovh-cds** [comment]
CDS Report build-monorepository#8155.0 ✘  
* 
  * Build ✘

File: packages/manager/apps/telecom/src/components/sidebar/sidebar.config.js [hunk 2]
```diff
@@ -8,11 +8,13 @@ import { ALIAS_AVAILABILITY } from '../../app/telecom/orders/alias/orders-alias.
...
 import { TELEPHONY_AVAILABILITY } from '../../app/telecom/telephony/telecom-telephony.constant';
 
+import { URLS } from './sidebar.constants';
+
 angular.module('managerApp').run(($translate, asyncLoader) => {
   asyncLoader.addTranslations(
     import(`./translations/Messages_${$translate.use()}.json`)
       .catch(() => import(`./translations/Messages_${$translate.fallbackLanguage()}.json`))
-      .then((x) => x.default),
+      .then(x => x.default),
```
[the remark this answers is not in the document]
Comment (marie-j) [reply packages/manager/apps/telecom/src/components/sidebar/sidebar.config.js]: I guess due to solving conflicts during rebase. Running eslint to fix

**Pull request merged**